views/index.ejs

```diff
@@ -69,6 +70,11 @@
 					<td><%= students[i].grade %></td>
 					<td><%= students[i].email %></td>
 					<td><%= students[i].id_number %></td>
+				<td>
+					<% for (let j = 0; j < students[i].program_list.length; j++) { %>
+						<a href="/view_program/<%= students[i].program_list[j].substring(students[i].program_list[j].indexOf("-") + 2) %>"><%= students[i].program_list[j].substring(0,students[i].program_list[j].indexOf("-") - 1) %></a>
```

Comment:
Is there a way to simplify this? I'm not sure exactly what it's doing. You can probably just show the program name here